docker-compose.yml

```diff
@@ -147,5 +147,17 @@ services:
       - databases
     tmpfs: /var/lib/mssql
 
+  mssql-2017:
```

Comment:
You reference 2019 in the makefile, but the compose has only 2017.

Reply:
Wait, it is here just above?

Reply:
Ah this was added in an earlier iteration already. No problem then!